docs/data/osdf/install-cache-rpm.md

```diff
@@ -0,0 +1,171 @@
+            if you cannot fit that much disk into a single chassis
+        * 24 GB of RAM
+    * A cache being used to serve data from the OSDF to a single site should have at least:
+        * 8 cores
+        * 40 Gbps connectivity
+        * 2 TB of NVMe disk for the cache partition
+        * 24 GB of RAM
+  We suggest that several gigabytes of local disk space be available for log files,
+  although some logging verbosity can be reduced.
+
+As with all OSG software installations, there are some one-time steps to prepare in advance:
+
+* Obtain root access to the host
+* Prepare [the required Yum repositories](../../common/yum.md)
+
+
+!!! note "OSG 23"
+    In OSG 23, the Pelican-based OSDF RPMs are only available in the "osg-upcoming" repositories.
+
+!!! note "Host certificates"
+    Caches should use a CA that is accepted by major browsers and operating systems,
+    such as InCommon RSA or [Let's Encrypt](../../security/host-certs/lets-encrypt).
+    IGTF certs are not recommended because clients are not configured to accept them by default.
+    Note that you will need the full certificate chain, not just the certificate.
+
+    The following locations should be used (note that they are in separate directories):
+    
+    * **Host Certificate Chain**: `/etc/pki/tls/certs/pelican.crt`
+    * **Host Key**: `/etc/pki/tls/private/pelican.key`
+
+
+Installing the Cache
+--------------------
+
+The cache service is provided by the `osdf-cache` RPM.
+Install it using one of the following commands:
+
+OSG 23:
+```console
+root@host # yum install --enablerepo=osg-upcoming osdf-cache
+```
+
+OSG 24:
+```console
+root@host # yum install osdf-cache
+```
+
+
+Configuring the Cache Server
+----------------------------
+
+!!! note "osdf-cache 7.11.1"
+    This configuration requires version 7.11.1 or newer of the `osdf-cache`
+    and `pelican` RPMs.
+    
+Configuration for a Pelican-based OSDF Cache is located in files in `/etc/pelican/config.d`.
+
+You must set the following config options:
+
+In `/etc/pelican/config.d/20-cache.yaml`, set `Cache.LocalRoot`, `Cache.DataLocation` and `Cache.MetaLocation` as follows,
+replacing `<CACHE PARTITION>` with the mount point of the partition you will use for the cache.
+```
+Cache:
+  LocalRoot: "<CACHE PARTITION>/namespaces"
+  DataLocation: "<CACHE PARTITION>/data"
+  MetaLocation: "<CACHE PARTITION>/meta"
+```
+
+
+Preparing for Initial Startup
+-----------------------------
+
+1.  The cache identifies itself to the federation via public key authentication;
+before starting the cache for the first time, it is generate a keypair.
+
+        :::console
+        root@host$ cd /etc/pelican
+        root@host$ pelican generate keygen
+
+
+    The newly created files, `issuer.jwk` and `issuer-pub.jwks` are the private and public keys, respectively.
+    **Save these files**; if you lose them, your cache will need to be re-approved.
```

Comment:
I think you intended for this last sentence to be the second step of the enumerated list?